### PR TITLE
Popup menu and dynamically adding everything

### PR DESCRIPTION
**Pull request by bhushan354**
- Generated projects section dynamically.
- Created popUp .
- Added popUp functionality .

Thank you ,
Bhushan .

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,3 +27,320 @@ hamburgerCloseConst.addEventListener('click', closeMenu);
...
+  iconImg.setAttribute('class', 'iconOnTop');
+
+  mainDivOne.appendChild(iconImg);
+
+  const mainDivTwo = document.createElement('div');
+  mainDivTwo.setAttribute('class', 'firstDiv');
+  mainDivOne.appendChild(mainDivTwo);
+
+  const mainText = document.createElement('h1');
+  mainText.textContent = project.name;
```
Comment (Ibizugbe) [inline app.js:114]: Nice job implementing the popup functionalities and dynamically generating your project section as required😉. but let's try to make the popup match the given design as this is a requirement for this milestone👍🏼.

yours

Screenshot 2023-07-26 at 16 19 27

design
Screenshot 2023-07-26 at 16 47 26

take note of the highlighted elements and add the necessary spacings around them👍🏼. for instance, you can see the cancel button is not aligned properly with the heading like it is in the design, and also the button icons are not well positioned😉. please take note of all the highlighted elements and make the necessary change.

NOTE: the same thing applies to your mobile screen as there are also some deviations from the design. Kindly take a look at the design and ensure everything matches both on mobile and desktop😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍🏼.
- [x] Pop up functionalities works properly👍🏼.
- [x] Project section is generated dynamically as required👍🏼.
- [x] Button interactions have been implemente👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] I suggest you create a folder for all your images and put them there as this makes your file structure more professional and makes your codebase esily accessible and maintainable.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bhushan354** [comment]
Hello sir as per previous reviewer suggested I have added changes accordingly hope this will pass all the checks .

Thank you ,
Bhushan.

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @bhushan354                 👋

Your project is complete! ✔️✔  🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors


- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**